packages/atoms/src/TipToast.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { TouchableOpacity } from 'react-native';
+
+import { Icon } from '@trezor/icons';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+import { Box } from './Box';
+import { Text } from './Text';
+
+type Props = {
```

Comment:
Please always use patter `ComponentNameProps`, it's easier when you will later decide to export it you don't need to rename it everywhere. I am actually thinking about exporting Props for all atoms by default.

Reply:
[32955a2](https://github.com/trezor/trezor-suite/pull/5530/commits/32955a29c0d6df4a1f26364c90ffe76921c322b7)